src/SprykerEco/Zed/Unzer/Business/Credentials/Validator/Constraints/TransferConstraintValidator.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+/**
+ * MIT License
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace SprykerEco\Zed\Unzer\Business\Credentials\Validator\Constraints;
+
+use Spryker\Shared\Kernel\Transfer\AbstractTransfer;
+use Symfony\Component\Validator\Constraint;
+use Symfony\Component\Validator\ConstraintValidator;
+use Symfony\Component\Validator\Exception\UnexpectedTypeException;
+use Symfony\Component\Validator\Exception\UnexpectedValueException;
+
+class TransferConstraintValidator extends ConstraintValidator
+{
+    /**
+     * @param mixed $value
+     * @param \Symfony\Component\Validator\Constraint $constraint
+     *
+     * @throws \Symfony\Component\Validator\Exception\UnexpectedTypeException
+     * @throws \Symfony\Component\Validator\Exception\UnexpectedValueException
+     *
+     * @return void
+     */
+    public function validate($value, Constraint $constraint): void
```

Comment:
please avoid constraint usage at the business layer
we restrict such cases since they bring too much magic